src/interfaces/token.spec.ts

```diff
@@ -0,0 +1,475 @@
+import { Contract } from "@ethersproject/contracts";
+
+import { Address, Asset, ChainId, SdkError, TokenInterface, TokenMetadata } from "..";
+import { CachedFetcher } from "../cache";
+import { Context } from "../context";
+import { assetStaticVaultV2Factory, balanceFactory, tokenFactory } from "../factories";
+import { EthAddress } from "../helpers";
+import { Yearn } from "../yearn";
+
+const getPriceUsdcMock = jest.fn();
```

Comment:
Not needed for this PR, just a comment for us all:

We should start to make setup files so we don't have to repeat all this logic in every file we encounter

Reply:
@FiboApe what do you mean? What's not needed? The mock?

Reply:
Oh I mean the change, it's not actually for you to do it on this PR, but it's something that we should start thinking about